src/Mod/Path/PathScripts/PathDeburr.py

```diff
@@ -30,7 +30,7 @@
 import PathScripts.PathOpTools as PathOpTools
 import math
 
-from PySide import QtCore
+from PySide import QtCore, QtGui
```

Comment:
Including `QtGui` in the op logic is kinda a no-no. This belongs into the ...Gui.py but not here.

Reply:
removed gui dependency.  Still using console messages.  Would be nice to throw the error and catch in the GUI but not sure how to do that if the taskpanel isn't open.